WooCommerce/Classes/Authentication/Prologue/LoginProloguePageViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+
+/// Displays the Login Prologue carousel, populated with `LoginProloguePageTypeViewController` pages.
+///
+final class LoginProloguePageViewController: UIPageViewController {
+
+    private var pages: [UIViewController] = []
+
+    private var pageControl: UIPageControl!
```

Comment:
Can we get rid of this implicitly unwrapped optional? Maybe turning this into a lazy variable?